README.md

```diff
@@ -220,7 +220,7 @@ The following table indicates which providers are supported on each platform. As
 | [network_snmp](#type-network_snmp) | ✅ | ✅ | ✅ | ❌ | ❌ | ❌ | ❌ |
 | [radius](#type-radius) | ✅ | ✅ | ✅ | ❌ | ❌ | ❌ | ❌ |
 | [radius_global](#type-radius_global) | ✅ | ✅ | ✅ | ❌ | ❌ | ❌ | ❌ |
-| [radius_server](#type-radius_server) | ✅ | ✅ | ✅ | ❌ | ❌ | ❌ | ❌ |
+| [radius_server](#type-radius_server) | ✅ | ✅ | ✅ | ❌ | ❌ | ❌ | ✅ |
```

Comment:
Sorry, one more thing I forgot - since there are a couple of properties that aren't supported on XR, you should add a `*` for the XR column and a link to a caveats table in the property description below - see any of the BGP types for an example.